src/webhookValidator.ts

```diff
@@ -70,18 +75,9 @@ export function webhookValidator({
     const rawBody = JSON.stringify(body);
     const hasBodyHash = originalUrl.includes('bodySHA256');
```

Comment:
I see the references to `bodySHA256` in the express webhook validator but I don't see it referenced in the security documentation. I also notice that `validateRequest` has a typescript declaration, but  `validateRequestWithBody` does not. The combination of those two things makes me think `bodySHA256` maybe shouldn't be used. How did you find out about that?

Reply:
This was added in a [PR](https://github.com/twilio/twilio-node/pull/365) in July of last year. Playing around with webhooks, I never encountered a `bodySHA256` query param, but they use it in their SDK so it felt like a good idea to adopt it for future proofing.